packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.Props.ts

```diff
@@ -102,3 +108,21 @@ export interface IChoiceGroupOption {
    */
   isDisabled?: boolean;
 }
+
+export interface IChoiceGroupField extends IChoiceGroupOption {
+  /**
+   * Boolean to allow onRenderOption access to state
+   */
+  checked?: boolean;
```

Comment:
they were, but then we deprecated them 

![image](https://cloud.githubusercontent.com/assets/1434956/26332581/774c8af0-3f0c-11e7-8590-702893e022db.png)